crypto/armcap.c

```diff
@@ -23,7 +24,7 @@ void OPENSSL_cpuid_setup(void)
 {
 }
 
-unsigned long OPENSSL_rdtsc(void)
+unsigned long long OPENSSL_rdtsc(void)
```

Comment:
I'm not convinced that making this always return `unsigned long long` is a good idea.  Each CPU will have a native _best_ size.